data/org.freedesktop.portal.InputCapture.xml

```diff
@@ -0,0 +1,547 @@
+
+        Create a capture input session. A successfully created session can at
+        any time be closed using org.freedesktop.portal.Session.Close(), or may
+        at any time be closed by the portal implementation, which will be
+        signalled via #org.freedesktop.portal.Session::Closed.
+
+        Supported keys in the @options vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>handle_token s</term>
+            <listitem><para>
+              A string that will be used as the last element of the @handle. Must be a valid
+              object path element. See the #org.freedesktop.portal.Request documentation for
+              more information about the @handle.
+            </para></listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>session_handle_token s</term>
+            <listitem><para>
+              A string that will be used as the last element of the session handle. Must be a valid
+              object path element. See the #org.freedesktop.portal.Session documentation for
+              more information about the session handle.
+            </para></listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>capabilities u</term>
+            <listitem><para>
+              Bitmask of requested capabilities, see the
+              SupportedCapabilities property. This value is required.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+
+        The following results get returned via the #org.freedesktop.portal.Request::Response signal:
+        <variablelist>
+          <varlistentry>
+            <term>session_handle o</term>
+            <listitem><para>
+              The session handle. An object path for the
+              #org.freedesktop.portal.Session object representing the created
+              session.
+            </para></listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>capabilities u</term>
+            <listitem><para>
+              The capabilities available to this session. This is always a
+              subset of the requested capabilities.
+              See the SupportedCapabilities property for details. Note that
+              while a capability may be available to a session, there is no
+              guarantee a device with that capability is currently available
+              or if one does become available that it will trigger input capture.
+
+              It is best to view this set as a negative confirmation - a
+              capability that was requested but is missing is an indication that
+              this application may not capture events of that capability.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+    -->
+    <method name="CreateSession">
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="handle" direction="out"/>
```

Comment:
We're expecting to show the dialog in response to this method being called, right? Since When we have the session, the app will be able to call things like GetZones() etc without any extra nagging.

With that assumption, we need a `parent_window` input argument, i.e. a
```
        @parent_window: Identifier for the application window, see <link linkend="parent_window">Common Conventions</link>
```

and 

```xml
      <arg type="s" name="parent_window" direction="in"/>
```

Reply:
done